### PR TITLE
Edits in nb2

### PR DESCRIPTION
**Pull request by fmaussion**
Changed:
- a comment in a cell (not important)
- the ELA experiment to run until equilibrium (600 years) because I find the resulting figure moire informative
- add a question about what similarity there is between the three glaciers in different ELA (bringing the students to notice that the area above/below ELA is always the same (50%), regardless of the size of the glacier
- added a MB profile pic I found somewhere, that might help with the discussion
- changed a bit of code that didn't run

### CODE REVIEW

**Review by ehultee — COMMENTED** [review]
Am I interpreting correctly that the change in what is now lines 313-321 is to make it more clear to students what we are manipulating and adding to the list models_MBG?

The MBG question...yes, maybe.  As previously discussed I find MBG un-intuitive, and so rather than a requirement, the last bit is more to have a suggested direction in which to play.  I can re-classify as "optional activity" or something, did that in another notebook.

Thanks!

File: 2_balance_de_masa.ipynb [hunk 1]
```diff
@@ -278,16 +308,17 @@
     "# equilibrium line altitude (ELA)\n",
     "ELA = 3000\n",
     "\n",
-    "for k, gradient in enumerate(grads):\n",
+    "for grad in grads:\n",
```
Comment (fmaussion) [inline 2_balance_de_masa.ipynb:313]: this is just a name change for consistency with below cells

**Comment by ehultee** [comment]
Actually: your comment in the MBG section is about the *code* question being difficult to realize, or it's about the tinkering needed to achieve a reduction by half in volume being harder conceptually than for the ELA (i.e. the science part is tough)?  I assumed the latter, but if the former, yeah I can add a comment that I'll demonstrate.

**Comment by fmaussion** [comment]
> Am I interpreting correctly that the change in what is now lines 313-321 is to make it more clear to students what we are manipulating and adding to the list models_MBG?

yes, and also indexing `models_MBG` isn't really necessary and a bit dangerous (since the list grows in the same loop, generally a bit bad practice). So altogether I find the code more intuitive after change, but again: just a subjective opinion.

**Comment by fmaussion** [comment]
> your comment in the MBG section is about the code question being difficult to realize, or it's about the tinkering needed to achieve a reduction by half in volume being harder conceptually than for the ELA (i.e. the science part is tough)? I assumed the latter, but if the former, yeah I can add a comment that I'll demonstrate.

yes I thought you meant code! All good then

**Comment by ehultee** [comment]
I did mean code, and I think the code is straightforward but the science is not.  Is that what you meant?  Or you meant you anticipate problems finding how to modify the code?

(It is always somewhat difficult to predict where people will have trouble, so this is interesting.)

**Comment by fmaussion** [comment]
If your students have no coding experience they will be totally overwhelmed by it - it's likely they'll do `shift+enter` and see what happens. But if you take your time with this notebook in particular (and probably you should), then OK!

**Comment by fmaussion** [comment]
I'll add one more comment in the code to clarify -> I'm on it

**Comment by fmaussion** [comment]
OK, I thing we are good here and on the same  page

**Pull request merged**